src/Style.js

```diff
@@ -291,7 +295,22 @@ class Style extends Component {
     doNotPrefix: PropTypes.bool,
     hasSourceMap: PropTypes.bool,
     id: PropTypes.string,
-    isMinified: PropTypes.bool
+    isMinified: PropTypes.bool,
+    autoPrefixerOpts: PropTypes.shape({
+      // shape defined by autoprefixer options documentation at https://www.npmjs.com/package/autoprefixer
+      browsers: PropTypes.array,
```

Comment:
array of what? might as well make it really specific, `PropTypes.arrayOf(PropTypes.string)`